app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -267,6 +268,10 @@ public void clearApplicationData(Context context, LogoutListener logoutListener)
         }
 
         sessionManager.logout()
+            .andThen(Completable.fromCallable(() -> {
+                updateAllDatabases();
+                return null;
```

Comment:
It is annoying we have to return null here, in kotlin the void return types analogue is Unit. You don't have to do any changes here I am pretty sure but be careful about null and Rx, it causes a crash in Single/Flowable/Observable since RxJava 2.0 with the introduction of Maybe to handle potentially null things

Reply:
Actually we can get rid of the null, d'oh. Use Callable.fromAction

Reply:
Done:)